src/main/java/org/example/controllers/PingController.java

```diff
@@ -0,0 +1,17 @@
+package org.example.controllers;
+
+
+
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@ResponseBody

```

Comment:
remove `@ResponeBody`, because `@RestController` contains this annotation already, check it yourself!